### PR TITLE
Add more redirects

### PR DESCRIPTION
**Pull request by bramkragten**

### CODE REVIEW

File: redirect.json [hunk 1]
```diff
@@ -26,14 +53,129 @@
...
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "general",
+    "description": "show your general Home Assistant settings",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "server_controls",
+    "description": "allow to control your server",
```
Comment (frenck) [inline redirect.json:112]: Lets avoid the word "allow", almost looks like the link reboot your machine or somethin

```suggestion
    "description": "Show your server controls",
```

File: redirect.json [hunk 2]
```diff
@@ -1,17 +1,44 @@
...
+  {
+    "redirect": "integrations",
+    "description": "show your integrations",
     "introduced": "2021.3"
   },
   {
-    "redirect": "logs",
-    "description": "show your Home Assistant logs",
+    "redirect": "config_flow_start",
+    "description": "allow starting a config flow",
```
Comment (frenck) [inline redirect.json:15]:
```suggestion
    "description": "start setting up a new integration",
```

File: redirect.json [hunk 3]
```diff
@@ -26,14 +53,129 @@
...
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "developer_states",
+    "description": "show to raw state of your entities",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "developer_services",
+    "description": "allow to call services",
```
Comment (frenck) [inline redirect.json:142]:
```suggestion
    "description": "show the call services in developer tools",
```

File: redirect.json [hunk 4]
```diff
@@ -26,14 +53,129 @@
...
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "developer_template",
+    "description": "allow to develop templates",
```
Comment (frenck) [inline redirect.json:147]:
```suggestion
    "description": "show template editor in developer tools",
```

File: redirect.json [hunk 5]
```diff
@@ -26,14 +53,129 @@
...
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "developer_events",
+    "description": "allow to listen and fire events",
```
Comment (frenck) [inline redirect.json:152]:
```suggestion
    "description": "Show debug tools for events",
```

File: redirect.json [hunk 6]
```diff
@@ -26,14 +53,129 @@
...
+    "description": "show debug tools for events",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "supervisor",
+    "description": "show your Supervisor dashboard",
+    "introduced": "supervisor-2021.02.10"
+  },
+  {
+    "redirect": "supervisor_system",
```
Comment (balloob) [inline redirect.json:161]: Let's split this into `supervisor_info` and `supervisor_logs`. Once we consolidate the logs, it won't be on 1 spot anymore.

**Review by balloob — APPROVED** [review]

**Pull request merged**